taps_to_cases/runner/src/main/java/com/microsoft/hydralab/t2c/runner/elements/IOSElementInfo.java

```diff
@@ -0,0 +1,107 @@
+package com.microsoft.hydralab.t2c.runner.elements;
+
+public class IOSElementInfo extends BaseElementInfo {
+    public IOSElementInfo(String xpath) {
+        super(xpath);
+    }
+
+    private String type;
+    private String name;
+    private String label;
+    private boolean enabled;
+    private boolean visible;
+    private boolean accessible;
+    private int x;
+    private int y;
+    private int width;
+    private int height;
+    private int index;
+
+    public String getType() {
```

Comment:
How about add `import lombok.Data;` and add `@Data` for the class? :)

Reply:
So far, all the element info classes are not suitable to be annotated as Data. We can refactor this structure in the future